File: 240503/5의 배수의 합/sum-of-specific-condition-multiple.js
const fs = require('fs');
const [a, b] = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let sum = 0;
const min = Math.min(a, b);
const max = Math.max(a, b);

for(let i = min; i <= max; i++){
    if(i % 5 === 0) sum += i;
}

console.log(sum);

File: 240503/5의 배수의 합/sum-of-specific-condition-multiple.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();

        int sum = 0;
        int max = Math.max(a, b);
        int min = Math.min(a, b);

        for(int i = min; i <= max; i++){
            if(i % 5 == 0){
                sum += i;
            }
        }

        System.out.println(sum);
    }
}